app/Check/Commands/SlackCheckStatusesCommand.php

```diff
@@ -69,18 +69,27 @@ protected function execute(
 		\Symfony\Component\Console\Input\InputInterface $input,
 		\Symfony\Component\Console\Output\OutputInterface $output
 	) {
+		$now = $this->dateTimeProvider->getDateTime();
+		$nowTime = $now->format("H:i");
 		$options = [
 			'paused' => FALSE,
 			'this->project->maintenance' => NULL,
 		];
+
 		$checks = $this->checksRepository->findBy($options);
 
 		foreach ($checks as $check) {
 
+			if ($nowTime >= $check->project->pausedFrom && $nowTime <= $check->project->pausedTo) {
+				continue;
+			}
+
 			$conditions = [
 				'check' => $check,
 			];
-			$locks = $this->slackNotifyLocksRepository->findBy($conditions)->orderBy('locked', \Nextras\Orm\Collection\ICollection::DESC)->limitBy(1);
+			$locks = $this->slackNotifyLocksRepository->findBy($conditions)->orderBy('locked',
```

Comment:
Tady je změna omylem kvůli automatickému přeformátování, tak to prosím vrať zpět